vhost-device-sound/src/stream.rs

```diff
@@ -239,85 +244,92 @@ impl Default for PcmParams {
     }
 }
 
-pub struct Buffer {
-    data_descriptor: virtio_queue::Descriptor,
+pub struct Request {
+    //data_descriptor: virtio_queue::Descriptor,
```

Comment:
why is this commented out? It is not described in the commit message of this commit.

Reply:
I forgot to remove this comment. Apologies. This field is not require since we can directly use the DescriptorChain from the IOMessage.